### DISCUSSION TITLE
Blog comment on Hurricane: Writing Blog Posts with Generative Feedback Loops

### DISCUSSION THREAD

**Discussion by giscus** [weaviate/weaviate-io#1861] [Docs/blog comments]
# Blog comment on Hurricane: Writing Blog Posts with Generative Feedback Loops

Hurricane is a web application to demonstrate Generative Feedback Loops with blog posts.

https://weaviate.io/blog/hurricane-generative-feedback-loops

**Comment by mikeldking** [comment]
Amazing post Connor!

**Comment by sj7272** [comment]
Love this post, but I couldn't get the code to work...
- No mentions of YDC_API_KEY
- No mentions of required libraries
- Backend won't run - shows {"detail":"Not Found"} on port 8000

Would be great if this/the repo could be updated
